project/ScoverageSettings.scala

```diff
@@ -22,7 +22,7 @@ object ScoverageSettings {
   def apply(): Seq[Def.Setting[_ >: String with Double with Boolean]] =
     Seq( // Semicolon-separated list of regexes matching classes to exclude
       ScoverageKeys.coverageExcludedPackages := "<empty>;.*Reverse.*;.*(config|testonly).*;.*(BuildInfo|Routes).*",
-      ScoverageKeys.coverageMinimum := 94.02,
+      ScoverageKeys.coverageMinimum := 94.06,
```

Comment:
Is 95 the upper limit for threshold ? or is it 100%?

Reply:
95% is the agreed target to work towards.